### PR TITLE
perf(inputs/influxdb_listener): benchmark serving writes

### PR DESCRIPTION
**Pull request by goller**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

Here are some benchmarks for the influxdb listener.  I'm hoping these can be guides for performance if there are refactors to do in the plugin.

### CODE REVIEW

File: plugins/inputs/influxdb_listener/influxdb_listener_test.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+					b.Error(err)
+				}
+				listener.serveWrite(res, req)
+				if res.Code != http.StatusNoContent {
+					b.Errorf("unexpected status %d", res.Code)
+				}
+			}
+		})
+	}
+	//}
```
Comment (reimda) [inline plugins/inputs/influxdb_listener/influxdb_listener_test.go]: remove stray comment

File: plugins/inputs/influxdb_listener/influxdb_listener_test.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		},
+		MaxBodySize: internal.Size{
+			Size: DEFAULT_MAX_BODY_SIZE,
+		},
+	}
+	listener.parser = influx.NewParser(listener.handler)
+	return listener
+}
+
+func BenchmarkHTTPListener_serveWrite(b *testing.B) {
```
Comment (reimda) [inline plugins/inputs/influxdb_listener/influxdb_listener_test.go]: Is the underscore normal here?  It looks kind of out of place.

Comment (goller) [reply plugins/inputs/influxdb_listener/influxdb_listener_test.go]: Hey @reimda I was following the format of Examples like:

```
 The naming convention to declare examples for the package, a function F, a type T and method M on type T are:

func Example() { ... }
func ExampleF() { ... }
func ExampleT() { ... }
func ExampleT_M() { ... }
```

**Comment by danielnelson** [comment]
@reimda Go ahead and merge when you are ready.

**Pull request merged**